app/code/Magento/Catalog/view/adminhtml/web/catalog/product/attribute/unique-validate.js

```diff
@@ -10,11 +10,17 @@ define([
     'use strict';
 
     return function (config) {
```

Comment:
@PieterCappelle  Here is a [doc](http://devdocs.magento.com/guides/v2.2/test/js/jasmine.html) and also you can see in dev/tests/js/jasmine/tests already created tests. If you will have some additional question contact me directly.